prov/rxm/src/rxm.h

```diff
@@ -127,9 +127,16 @@ struct rxm_mr {
 	struct fid_mr *msg_mr;
 };
 
+struct rxm_ep_wire_proto {
+	uint8_t	ctrl_version;
+	size_t inject_size;
+	size_t sar_limit;
+};
+
 struct rxm_cm_data {
 	struct sockaddr name;
 	uint64_t conn_id;
+	struct rxm_ep_wire_proto proto;
 };
```

Comment:
Is this a wire structure, or only used locally (except for the proto field)?  sockaddr name prevents support for ipv6.

Reply:
This (rxm_cm_data ) is used both locally and remotely

Reply:
Does rxm_cm_data actually get placed on the wire?  if it does, then we should allow for ipv6 support, plus ensure that all fields are properly aligned.  if not, then we can make adjustments later as needed.